python/cudf_polars/cudf_polars/experimental/benchmarks/utils.py

```diff
@@ -196,6 +198,12 @@ def summarize(self) -> None:
                     f"mean time: {np.mean([record.duration for record in records]):0.4f}"
                 )
                 print("=======================================")
+        total_mean_time = sum(
+            np.mean([record.duration for record in records])
```

Comment:
You can use `statistics.mean` from the standard library to avoid the numpy import